theme/brb/views/authentication/home.hbs

```diff
@@ -0,0 +1,33 @@
+{{#extend "lib/default"}} {{#content "styles" mode="append"}}
+<link rel="stylesheet" href="/styles/authentication/home.css" /> {{/content}} {{#content "scripts" mode="append"}}
+<script src="/scripts/login.js" type="text/javascript" defer></script>
+{{/content}} {{#content "body"}} {{#embed "lib/header" hideMenu=true}}{{/embed}}
+<div class="container">
+    <div class="row">
+        <main class="col-md-8">
+            <div class="hidden-md-up">
+                {{> "authentication/forms/login"}} {{> "authentication/forms/demo-login"}}
+            </div>
+
+            <section class="card">
+                <div class="card-block">
+                    <h4>Herzlich Willkommen bei der Schul-Cloud Brandenburg!</h4>
+                    <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.   
```

Comment:
seems not to be ready...

Reply:
Startseite, About Seite und andere kleinere Textstellen folgen Montag als Fix. Das ist jetzt erstmal nur Technik-Test